imports/modules/Space/server/index.js

```diff
@@ -111,6 +111,15 @@ Meteor.publish('spaceEvents', function(hex) {
   check(hex, Object);
 
   if (this.userId) {
+    const subsCounts = {};
```

Comment:
subsCounts = Object.values(this._session._namedSubs).reduce((sum, x) =>sum += x._name === this._name ? 1 : 0, 0)